velox/dwio/dwrf/common/Compression.h

```diff
@@ -84,9 +85,17 @@ static std::unique_ptr<dwio::common::BufferedOutputStream> createCompressor(
       encrypter);
 }
 
-static const CompressionOptions getDwrfOrcDecompressionOptions() {
+static const CompressionOptions getDwrfOrcDecompressionOptions(
```

Comment:
No `static` function in header file, change it to `inline`. Also there is no need to `const` rvalue returned